GOVERNANCE.md

```diff
@@ -0,0 +1,101 @@
+
+A Maintainer who shows an above-average level of contribution to the project, particularly with respect to its strategic direction and long-term health, may be nominated to become an Owner, described below.
+
+### Owners
+The WP Rig project is governed by the [Owner group](#list-of-current-owners). They are collectively responsible for high-level guidance of the project.
+
+The [Owner group](#list-of-current-owners) has final authority over this project including:
+
+- Technical direction of the project, especially infrastructure PRs and linting and/or schema decisions.
+- Project governance and process (including this policy and any updates).
+- Contribution policy.
+- GitHub repository hosting.
+
+Being an Owner is not time-limited. There is no fixed size of the Owner group. The Owner group should be of such a size as to ensure adequate coverage of important areas of expertise balanced with the ability to make decisions efficiently.
+An Owner may be removed from the Owner group by voluntary resignation, or by a standard Owner group motion, including for violations of Code of Conduct.
+
+Changes to the Owner group should be posted in the agenda, and may be suggested as any other agenda item (see [Project Meetings](#project-meetings) below).
+
+Owners fulfill all requirements of Maintainers, and also:
+
+- Ensure the smooth running of the project.
+- Approve changes to this document.
+- Manage and merge commits to the `master` branch.
+
+To become an Owner one must fulfill at least the following conditions and commit to being a part of the community for the long-term.
+
+- Have worked in a helpful and collaborative way with the WordPress and WP Rig communities.
+- Have given good feedback on others’ submissions and displayed an overall understanding of the code quality standards for the project.
+- Have the ability to drive the project forward, manage requirements from users, and taking on responsibility for the overall health of the project.
+
+An individual is invited to become an Owner by existing Owners. A nomination will result in discussion and then a decision by the Owner group.
+
+#### List of current Owners
+- Morten Rand-Hendriksen (@mor10)
+- Rachel Cherry (@bamadesigner)
+
+## Decision Making
+Decision making generally follows a [Consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making) model.
+
+When an agenda item has appeared to reach a consensus, the moderator will ask “Does anyone object?” as a final call for dissent from the consensus.
+
+If an agenda item cannot reach a consensus, an owner can call for either a closing vote or a vote to table the issue to the next meeting. The call for a vote must be approved by a majority of the owners or else the discussion will continue. Simple majority wins.
+
+## Project Meetings
+There are bi-weekly project meetings; they are scheduled in a shared calendar, and using tools that enable participation by the community. 
```

Comment:
Should we link to the shared calendar here? https://calendar.google.com/calendar/embed?src=wprigio%40gmail.com&ctz=America%2FChicago